build/gulp/tasks/docs.ts

```diff
@@ -87,6 +91,7 @@ task('build:docs:component-menu-behaviors', () =>
 
 task('build:docs:example-menu', () =>
   src(examplesSrc, { since: lastRun('build:docs:example-menu') })
+    .pipe(remember('example-menu')) // FIXME: with watch this unnecessarily processes index files for all examples
```

Comment:
another problem that I might see here is that if file will be removed, it will still remain to be part of the stream - and this will result in non-relevant produced result.

Reply:
https://github.com/stardust-ui/react/pull/187/files#diff-d0223606702a9bcfa799433ee11ed0aeR26

Reply:
for some reason this 'unlink' event doesn't fire on my machine - seems to be an issue with 'chokidar' lib (while for 'change' events everything works). This was, essentially, the reason to ask - thanks!